### PR TITLE
Add aplpy to conda-forge

### PR DESCRIPTION
**Pull request by mwcraig**
@astrofrog -- are you ok with being a maintainer on this?

### CODE REVIEW

**Comment by mwcraig** [comment]
Restarting circleCI because it looked like an unrelated network issue...

**Comment by bsipocz** [comment]
Sure, I'm happy to be on any feedstock @mwcraig adds.

**Comment by synapticarbors** [comment]
@ocefpaf -- Did we get the ok from @astrofrog? I didn't see it although perhaps a blanket permission was given in another PR.

**Comment by ocefpaf** [comment]
> @ocefpaf -- Did we get the ok from @astrofrog? I didn't see it although perhaps a blanket permission was given in another PR. 

@astrofrog is a conda-forge veteran and knows what to do. Don't worry :wink:

**Comment by synapticarbors** [comment]
@ocefpaf it wasn't really a question of whether someone listed knows what to do for me. It's about making sure people don't get added to projects they didn't explicitly consent to being a part of (whether by replying on a PR or having the person who issued the PR acknowledge that they've conferred with the person beforehand). I prefer to err on the side of waiting even if I know they are maintaining other projects. 

I think this is a reasonable position and one that maybe should be codified somewhere official as a community reviewer's standard. 

/cc @jakirkham

**Comment by ocefpaf** [comment]
> I prefer to err on the side of waiting even if I know they are maintaining other projects.

And you are correct to do so. I merged this b/c I know the people involved and I know that @astrofrog can just remove himself if he does not want to be a maintainer.

TL;DR don't do that, but don't fret too much if others do :wink:

**Comment by mwcraig** [comment]
I should have spoken up earlier -- @astrofrog is the primary maintainer of APLpy and had previously agreed to be listed as s maintainer on the astropy related packages that he is maintainer on.

I do agree we need a recipe reviewer guide....I made a small start at https://github.com/conda-forge/staged-recipes/wiki/Reviewing-recipes-on-conda-forge-staged-recipes but if you could add to it @synapticarbors that would be great.

At this point even just adding questions would be helpful (I should do that too)...
